ci/linux_mips64el

```diff
@@ -0,0 +1,7 @@
+FROM multiarch/debian-bootstrap:mips64el-bullseye
```

Comment:
You probably want buster not bullseye? buster is current stable, bullseye is current testing.